### PR TITLE
[BPK-816] Adjust Text docs wording

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: packages/bpk-docs/src/pages/TypographyPage/TypographyPage.js [hunk 1]
```diff
@@ -45,9 +45,9 @@ const components = [
     title: 'Text',
     blurb: [
       <Paragraph>
-        <BpkCode>BpkText</BpkCode> is the one stop shop for your text needs. It enables you to use the six backpack
-        defined text styles in combination with any of the heading tags, a span or a paragraph. You can mix different
-        text styles with the appropriate tag to achieve semantic markup while retaining control over how the text looks.
+        The Text componnet enables you to use the six backpack defined text styles in combination with any of the
```
Comment (matthewdavidson) [inline packages/bpk-docs/src/pages/TypographyPage/TypographyPage.js]: ~componnet~ component

**Comment by matthewdavidson** [comment]
@k0nserv also noticed on http://backpack.prod.aws.skyscnr.com/components/atoms/typography that ~Paragaphs~ should be _Paragraphs_ :)

**Comment by k0nserv** [comment]
Fixed all comments

**Comment by matthewdavidson** [comment]
LGTM

**Pull request merged**